coral/README.md

```diff
@@ -2,34 +2,65 @@
 
 - Please be aware of our [Code of Conduct](../CODE_OF_CONDUCT.md) ❤️
 
+## Table of content
+
+* [About](#about)
+* [Installation and usage](#installation-and-usage)
+  + [Usage: How to run the frontend app in development](#usage--how-to-run-the-frontend-app-in-development)
+    - [Scripts used and what they execute](#scripts-used-and-what-they-execute)
+  + [Usage: How to run the Coral in build](#usage--how-to-run-the-coral-in-build)
+* [Tech stack](#tech-stack)
+  + [App development](#app-development)
+  + [Testing](#testing)
+  + [Linting and code formatting](#linting-and-code-formatting)
+* [Styling](#styling)
+* [Documentation](#documentation)
+  + [Tip](#tip)
+* [More detailed documentation](#more-detailed-documentation)
+
 ## About
 
-`/coral` contains a React app. It's the rewrite of the existing Klaw frontend. 
+`/coral` contains a React app. It's the rewrite of the existing Klaw frontend.
 
-## Installation and usage 
+## Installation and usage
 
-ℹ️ Coral uses `pnpm run` as a package manager. Read in their official documentation [how to install](https://pnpm.io/installation) it. 
+ℹ️ Coral uses `pnpm run` as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) it.
 
 - navigate to this directory
 - run `pnpm install`
-- run `pnpm dev` to start the frontend app in development mode
+- run `pnpm dev` to start the frontend app in development mode _with remote API_
+- run `pnpm dev-without-api` to start the frontend app in development mode without api
+
+ℹ️ **Using a remote API**
+We recommend doing coral development with a remote API.
+Please see our documentation: [Development with remote API](docs/development-with-remote-api.md).
+
+ℹ️ **Developing without remote API**
+If you want to run Coral without an API, you can do that, too.
+Please see our documentation: [Mocking an API for development](docs/mock-api-for-development.md)
 
-### Usage: How to run the app
+### Usage: How to run the frontend app in development
 
 ℹ️ You can see all our scripts in the [`package.json`](package.json).
 You can also run `pnpm` in your console to get a list of all available scripts.
 
 #### Scripts used and what they execute
 - `build`: builds the frontend app for production
 - `dev`: starts the frontend app for development
-- `lint`: runs a code format check and if no error is found, lints the code. 
+- `lint`: runs a code format check and if no error is found, lints the code.
   - the linting script does not mutate your code. See [Linting and code formatting](#linting-and-code-formatting) for more info.
 - `preview`: builds a preview production build _locally_
 - `reformat`: runs the code formatter (prettier) and reformat all code
 - `test-dev`: runs all test tests related to changed files in a watch mode
 - `test`: runs all tests one time
 - `tsc`: runs the TypeScript compiler
 
+### Usage: How to run the Coral in the build
```

Comment:
Nitpick:  "in the build" seems a little vague. Maybe we could be more specific?

Reply:
Good idea! I was not sure how to call it best, maybe something like: 

- How to run the Klaw project (or application?) with Coral
- How to run Coral inside the Klaw project (or application?) 
... 

That's it. That's all my ideas 🙈

Reply:
Also updated!